train.py

```diff
@@ -44,7 +44,14 @@
     from apex.parallel import convert_syncbn_model
     has_apex = True
 except ImportError:
-    has_apex = False
+    if 'moreh' not in torch.__version__:
+        has_apex = False
+    else:
+        apex = torch.moreh.apex
+        amp = apex.amp
+        ApexDDP = apex.parallel.DistributedDataParallel
```

Comment:
Since we are not going to manually use `distribute` option, this won't be necessary.

Reply:
I added these classes for the sake of completeness. They won't be used anyways.